.ci/install_kata_kernel.sh

```diff
@@ -13,80 +13,75 @@ set -e
 cidir=$(dirname "$0")
 source "${cidir}/lib.sh"
 
-repo_owner="clearcontainers"
-repo_name="linux"
-
-linux_releases_url="https://github.com/${repo_owner}/${repo_name}/releases"
-#fake repository dir to query kernel version from remote
-fake_repo_dir=$(mktemp -t -d kata-kernel.XXXX)
-
-function cleanup {
-	rm  -rf "${fake_repo_dir}"
+repo_name="packaging"
+repo_owner="kata-containers"
+kata_kernel_dir="/usr/share/kata-containers"
+kernel_arch="$(arch)"
+get_kernel_url="https://cdn.kernel.org/pub/linux/kernel"
+tmp_dir=$(mktemp -d)
+
+download_repo() {
+	pushd ${tmp_dir}
+	git clone --depth 1 https://github.com/${repo_owner}/${repo_name}
+	popd
 }
 
-trap cleanup EXIT
-
-function usage() {
-	cat << EOT
-Usage: $0 <version>
-Install the containers clear kernel image <version> from "${repo_owner}/${repo_name}".
-
-version: Use 'latest' to pull latest kernel or a version from "${cc_linux_releases_url}"
-EOT
-
-	exit 1
+get_kernel_version() {
+	pushd $tmp_dir/$repo_name > /dev/null
+	kernel_version=$(grep "Linux/[${kernel_arch}]*" kernel/configs/[${kernel_arch}]*_kata_kvm_* | cut -d' ' -f3 | tail -1)
```

Comment:
Thought: we might want to make `kvm` a variable since it's used twice and we will want to support other systems in the future.